src/route/route.js

```diff
@@ -287,17 +293,17 @@
 
           if ( !intendedRoute ) {
             if ( options.controller && !options.action ) { // we're just moving to a new controller and accepting the default action
-              intendedRoute = _.find( $route.routes, function( r ) {
+              intendedRoute = values( $route.routes ).filter( function( r ) {

```

Comment:
`find` isn't available in IE11 but `filter` is, so we filter then take the first (`[0]`) a few lines below